docs/source/json-api/index.rst

```diff
@@ -165,7 +138,14 @@ token.  The default "header" is fine.  Under "Payload", fill in:
       }
     }
 
-Keep in mind that the value of the ``ledgerId`` payload field has to match the one passed to the sandbox with the ``--ledgerid`` argument. You can replace ``Alice`` with whatever party you want to use.
+The value of the ``ledgerId`` field has to match the ``ledgerId` of your underlying DAML Ledger.
+For the ``daml sandbox`` this corresponds to the ``--ledgerid MyLedger`` flag.
+
+.. note:: The value of ``applicationId`` can currently be anything and does not need to correspond to ``daml json-api --application-id <value>``.
+
+The value for ``actAs`` is specified as a list. You can replace ``Alice`` with whatever party you want to use,
+provided that it is a valid party on the ledger.
+For the ``daml sandbox`` any string will create a party with that name.
 
 Under "Verify Signature", put ``secret`` as the secret (*not* base64
```

Comment:
This is a lie. It doesn’t matter what secret you use here. The JSON API only decodes the token. It doesn’t validate it which is the part where the secret would matter.